webapp/src/actions/index.ts

```diff
@@ -426,8 +426,10 @@ export function handleConnectFlow(instanceID?: string) {
 
 export function redirectConnect(instanceID: string) {
     return async (dispatch, getState) => {
+        const serverUrl = getPluginServerRoute(getState());
         const instancePrefix = '/instance/' + btoa(instanceID);
-        const target = '/plugins/' + PluginId + instancePrefix + '/user/connect';
+        const target = serverUrl + instancePrefix + '/user/connect';
```

Comment:
I think it might be simpler to maintain to use strings format here? Also to be more cohesive with the changes below, where you are using string format.

```typescript
const target = `${serverUrl}${instancePrefix}/user/connect`;
```